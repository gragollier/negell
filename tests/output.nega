int main() {
    output 1;
    output "hello";
    output false;
}