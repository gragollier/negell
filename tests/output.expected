1
hello
0